src/scss/skin-modern/components/_label.scss

```diff
@@ -9,6 +9,10 @@
   white-space: nowrap;
 }
 
-.#{$prefix}-ui-label {
+
+.#{$prefix}-ui-label-savedEnergy {
```

Comment:
We are changing the existing style for `.#{$prefix}-ui-label` so the style for existing labels style changes. Should we not instead define a new one?